sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -620,6 +623,219 @@ class HiveDDLSuite
     }
   }
 
+  test("CREATE TABLE LIKE a temporary table") {
+    val sourceTabName = "tab1"
+    val targetTabName = "tab2"
+    withTable(sourceTabName, targetTabName) {
+      spark.range(10).select('id as 'a, 'id as 'b, 'id as 'c, 'id as 'd)
+        .createTempView(sourceTabName)
+      sql(s"CREATE TABLE $targetTabName LIKE $sourceTabName")
+
+      val sourceTable =
+        spark.sessionState.catalog.getTableMetadata(TableIdentifier(sourceTabName, None))
+      val targetTable =
+        spark.sessionState.catalog.getTableMetadata(TableIdentifier(targetTabName, Some("default")))
+
+      assert(targetTable.storage.serde ==
```

Comment:
Since the source table is temporary view, we are unable to get the serde from the source table and thus we choose the default one. Here, it is for confirming that.